lib/request_headers_logger/text_formatter.rb

```diff
@@ -7,11 +7,22 @@ module TextFormatter
     def call(severity, time, progname, msg)
       format(::Logger::Formatter::Format,
              severity_name(severity)[0],
-             format_datetime(time),
+             format_time(time),
              $PID,
              severity_name(severity),
              progname,
-             msg2str("#{tags_text}#{msg}"))
+             "#{tags_text}#{to_string(msg)}")
+    end
+
+    def format_time(time)
+      return format_datetime(time) if defined?(format_datetime)
```

Comment:
Maybe `respond_to? ` is a bit more idiomatic?

Reply:
you mean something like this 
```
return format_datetime(time) if respond_to?(format_datetime)
```